manager/orchestrator/update/updater.go

```diff
@@ -406,7 +406,11 @@ func (u *Updater) updateTask(ctx context.Context, slot orchestrator.Slot, update
 	}
 
 	if delayStartCh != nil {
```

Comment:
Maybe the same should be done at https://github.com/docker/swarmkit/blob/master/manager/orchestrator/update/updater.go#L459